### PR TITLE
fix(node_benchmark): correctly count average if metric is missing

### PR DESCRIPTION
**Pull request by aleksbykov**
Fixes: #5751. If any node performance tool doesn't correclty store result in es document, next counting Average() value could lead to exception DivisionByZero, because final list is empty for failed metrics.

use list with single element [0.0] to avoid such collisions.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fgelcer — DISMISSED** [review]
LGTM, but it would be nice if instead of writing `Fix for issue #5751` you would use `Fixes: #5751`, so once this PR will be merged, it will close the issue automatically

**Review by soyacz — APPROVED** [review]
this will prevent from getting DivisionByZero exceptions but will show broken values.
Anyway we should fix other issues related this code (like fixing the reason why we don't get these values).
Besides, why `0.0` and not `float('nan')`?

**Comment by aleksbykov** [comment]
> Besides, why `0.0` and not `float('nan')`
in this case sum() will fail and class ComparableResult use 0.0

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
Please see why checks are failing and we can merge.

**Comment by fgelcer** [comment]
> Please see why checks are failing and we can merge.

something terminated the instance running the checks:
```
pylint...................................................................Cannot contact i-07159072a45c149e2: java.lang.InterruptedException
/bin/bash: line 1:    10 Killed                  ./sct.py pre-commit
```

i have re-run it

**Pull request merged**
